Refit.Tests/ResponseTests.cs

```diff
@@ -139,6 +139,43 @@ public async Task WhenProblemDetailsResponseContainsExtensions_ShouldHydrateExte
                 kvp => Assert.Equal(new KeyValuePair<string, object>(nameof(expectedContent.Baz), expectedContent.Baz), kvp));
         }
 
+        [Fact]
+        public async Task WithNonSeekableStream_UsingSystemTextJsonContentSerializer()
```

Comment:
What's the bit of the test that triggers the behaviour to go through the path that caused the exception?

Reply:
None yet, see my comment below 😄

I think I'll follow your advice and try inheriting from `MemoryStream` for the content.